drivers/net/wireless/mediatek/mt76/mt76_connac.h

```diff
@@ -134,8 +134,12 @@ mt76_connac_pm_unref(struct mt76_connac_pm *pm)
 static inline bool
 mt76_connac_skip_fw_pmctrl(struct mt76_phy *phy, struct mt76_connac_pm *pm)
 {
+	struct mt76_dev *dev = phy->dev;
 	bool ret;
 
+	if (mt76_is_mmio(dev) && dev->token_count)
```

Comment:
@objelf I do not think we need the mt76_is_mmio() check is token_count is increased/decreased by mmio device only. I will remove the check merging the patch

Reply:
ok, thanks.